cmd/ntpcheck/cmd/utils.go

```diff
@@ -257,7 +322,10 @@ func init() {
 	utilsCmd.AddCommand(ntpdateCmd)
 	ntpdateCmd.Flags().StringVarP(&remoteServerAddr, "server", "s", "", "Server to query")
 	ntpdateCmd.Flags().IntVarP(&remoteServerPort, "port", "p", 123, "Port of the remote server")
-	ntpdateCmd.Flags().IntVarP(&ntpdateRequests, "requests", "r", 3, "How many requests to send")
+	ntpdateCmd.Flags().IntVarP(&ntpdateRequests, "requests", "r", 1, "How many requests to send")
```

Comment:
Why are you changing defaults? This was implemented to mimic defaults of a real ntpdate which sends several packets.

Reply:
Fair point. SNTP clients send a single request (and I had that in mind) but happy to keep default at 3.